scripts/host-display

```diff
@@ -8,108 +8,96 @@ n.b. the value returned by the "status" command will be correct as of the next
 boot (assuming that the boot config is not modified further), and is not
 necessarily correct at the time this script is called.
 """
-from __future__ import print_function
 
 import os
-import os.path
 import re
-import subprocess
 import sys
 import syslog
+from pathlib import Path
+
 from xcp.bootloader import Bootloader
+from xcp.cmd import runCmd
+
+SHORT_COMMAND_NAME = Path(__file__).name
 
-COMMAND_NAME = sys.argv[0]
-SHORT_COMMAND_NAME = os.path.basename(COMMAND_NAME)
+XE_SERIAL = "xe-serial"
+XEN_CMDLINE = "/opt/xensource/libexec/xen-cmdline"
 
-XE_SERIAL = 'xe-serial'
-XEN_CMDLINE = '/opt/xensource/libexec/xen-cmdline'
 
 def send_to_syslog(msg):
-    """ Send a message to syslog. """
+    """Send a message to syslog."""
     pid = os.getpid()
-    syslog.syslog("%s[%d] - %s" %(SHORT_COMMAND_NAME, pid, msg))
-
-def doexec(args):
-    """Execute a subprocess, then return its return code, stdout and stderr"""
-    send_to_syslog(args)
-    proc = subprocess.Popen(
-        args,
-        stdin=None,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        close_fds=True,
-        universal_newlines=True)
-    rc = proc.wait()
-    stdout = proc.stdout
-    stderr = proc.stderr
-    return (rc, stdout, stderr)
```

Comment:
By replacing calls to `doexec()` (which calls send_to_syslog) with `xcp.runCmd()` (which does not log the commands to syslog), you remove the logging of the executed commands to the syslog.

Please revert the changes to call `xcp.runCmd()` directly and incorporate `syslog.syslog("%s[%d] - %s" % (SHORT_COMMAND_NAME, os.getpid(), msg))` from the now unused `send_to_syslog()` into the updated `doexec()`.

Reply:
updated and tested with job id 3889671